modules/table/src/main/java/org/apache/ignite/internal/table/IgniteTablesInternal.java

```diff
@@ -17,9 +17,7 @@
 
 package org.apache.ignite.internal.table;
 
-import java.util.List;
```

Comment:
It was also expected that you'll remove `org.apache.ignite.internal.table.distributed.storage.InternalTableImpl#assignments`
Please check https://github.com/apache/ignite-3/pull/2848/files#diff-e319264cb2e94af03c766ec91510f68ba83e69cc93873b8e214e9289f52bb67a

Reply:
1. #2848 is not yet merged
2. The method is used by `ItTableScanTest`

I'd like to avoid inflating this PR further, separate ticket created: https://issues.apache.org/jira/browse/IGNITE-20933